src/backend/templates/event.hbs

```diff
@@ -32,10 +32,11 @@
     </head>
 
     <body onload="datafetcher()">
+      <a href="#content" class="skip">Skip to content</a>
     {{> scroll}}
     <a id="top"></a>
     {{> navbar}}
-    <div class="carousel slide" style="margin-bottom: 0;">
+    <div class="carousel slide" style="margin-bottom: 0;" id="content">
```

Comment:
@anu-007 The point is ` Don't make the id same for all sections `  Like in your case you have used content in all files.